src/core_atmosphere/physics/Makefile

```diff
@@ -53,7 +53,7 @@ core_physics_mmm: core_physics_init
 	(cd physics_mmm; $(MAKE) all)
 
 core_physics_wrf: core_physics_init core_physics_mmm
-	(cd physics_wrf; $(MAKE) all COREDEF="$(COREDEF)")
+	(cd physics_wrf; cp ./GSL_cloud_physics/* .; $(MAKE) all COREDEF="$(COREDEF)")
```

Comment:
Currently the submodule files are copied to physics_wrf before compiling, but this should be improved, so that the submodule files are compiled in the submodule directory and linked appropriately.

Reply:
@jderrico-noaa This is outdated now. I added a Makefile in my last commit.